packages/ckeditor5-find-and-replace/src/findandreplaceui.js

```diff
@@ -122,6 +122,9 @@ export default class FindAndReplaceUI extends Plugin {
 				cancelEvent();
 			} );
 
+			// Dropdown won't be enabled when find command is disabled.
```

Comment:
This comment describes obvious thing - it basically reads the code one line below which makes it redundant. If anything we should say **why** we are doing this binding. You could do one sentence explanation and add a reference number to issue #10001 - this will give the most insights.